### PR TITLE
Added Social distance Parrot

### PR DESCRIPTION
**Pull request by ryusei-koseki**
Kakapo want a social distance.

### CODE REVIEW

**Comment by ryusei-koseki** [comment]
there was no parrots.json...

**Comment by ryusei-koseki** [comment]
I don't know what to do

File: parrots.yaml [hunk 1]
```diff
@@ -427,3 +427,5 @@
 - hd: hd/daftpunkparrot.gif
   name: Daft Punk Parrot
   tip: Harder Better Faster Stronger
+  - hd: hd/socialdistanceparrot.gif
```
Comment (urielsalis) [inline parrots.yaml:430]: this and the following line should be aligned in indentation with the previous one, you have a extra tab 😓

**Comment by urielsalis** [comment]
By the way, this already exists https://github.com/jmhobbs/cultofthepartyparrot.com/blob/master/parrots/docparrot.gif

**Comment by ryusei-koseki** [comment]
I see...

**Comment by jmhobbs** [comment]
Ack! Forgot to update the readme about the change to `parrots.yaml`, I'll do that.

We do already have a doctor parrot with the face mask, and this one seems to be sped up somehow.

I'm going to close for now, but thank you for submitting it and pointing out the issue with the instructions!

**Pull request closed without merging**